sites/ademe/src/backend/ademe/settings.py

```diff
@@ -0,0 +1,719 @@
+    ./configurations and compose the Base configuration with those mixins.
+
+    It depends on an environment variable that SHOULD be defined:
+
+    * DJANGO_SECRET_KEY
+
+    You may also want to override default configuration by setting the following environment
+    variables:
+
+    * DJANGO_SENTRY_DSN
+    * RICHIE_ES_HOST
+    * DB_NAME
+    * DB_USER
+    * DB_PASSWORD
+    * DB_HOST
+    * DB_PORT
+    """
+
+    DEBUG = False
+
+    SITE_ID = 1
+
+    # Security
+    ALLOWED_HOSTS = values.ListValue([])
+    SECRET_KEY = "ThisIsAnExampleKeyForDevPurposeOnly"  # nosec
+    # System check reference:
+    # https://docs.djangoproject.com/en/3.1/ref/checks/#security
+    SILENCED_SYSTEM_CHECKS = values.ListValue(
+        [
+            # Allow the X_FRAME_OPTIONS to be set to "SAMEORIGIN"
+            "security.W019"
+        ]
+    )
+    # The X_FRAME_OPTIONS value should be set to "SAMEORIGIN" to display
+    # DjangoCMS frontend admin frames. Dockerflow raises a system check security
+    # warning with this setting, one should add "security.W019" to the
+    # SILENCED_SYSTEM_CHECKS setting (see above).
+    X_FRAME_OPTIONS = "SAMEORIGIN"
+
+    # Application definition
+    ROOT_URLCONF = "ademe.urls"
+    WSGI_APPLICATION = "ademe.wsgi.application"
+
+    # Database
+    DATABASES = {
+        "default": {
+            "ENGINE": values.Value(
+                "django.db.backends.postgresql_psycopg2",
+                environ_name="DB_ENGINE",
+                environ_prefix=None,
+            ),
+            "NAME": values.Value("ademe", environ_name="DB_NAME", environ_prefix=None),
+            "USER": values.Value("ademe", environ_name="DB_USER", environ_prefix=None),
+            "PASSWORD": values.Value(
+                "pass", environ_name="DB_PASSWORD", environ_prefix=None
+            ),
+            "HOST": values.Value(
+                "localhost", environ_name="DB_HOST", environ_prefix=None
+            ),
+            "PORT": values.Value(5432, environ_name="DB_PORT", environ_prefix=None),
+        }
+    }
+    MIGRATION_MODULES = {}
+
+    # Static files (CSS, JavaScript, Images)
+    STATIC_URL = "/static/"
+    MEDIA_URL = "/media/"
+    MEDIA_ROOT = os.path.join(DATA_DIR, "media")
+    STATIC_ROOT = os.path.join(DATA_DIR, "static")
+
+    # For static files, we want to use a backend that includes a hash in
+    # the filename, that is calculated from the file content, so that browsers always
+    # get the updated version of each file.
+    STATICFILES_STORAGE = values.Value("base.storage.CDNManifestStaticFilesStorage")
+
+    AUTHENTICATION_BACKENDS = ("django.contrib.auth.backends.ModelBackend",)
+
+    # AUTHENTICATION DELEGATION
+    RICHIE_AUTHENTICATION_DELEGATION = {
+        "BASE_URL": values.Value(
+            "", environ_name="AUTHENTICATION_BASE_URL", environ_prefix=None
+        ),
+        "BACKEND": values.Value(
+            "openedx-hawthorn",
```

Comment:
@jbpenrath Ah

Reply:
@sveetch so we must put
```suggestion
            "openedx-dogwod",
```

Reply:
I named the backend with the oldest openedx version supported : 
So `openedx-hawthorn` supports hawthorn, ironwood, juniper and koa (tested on hawthorn, and ironwood, not quit sure for koa and never tested on juniper)
And `openedx-dogwood` supports Dogwood, Eucalyptus. (tested on both)